projects/packages/my-jetpack/_inc/components/jetpack-manage-banner/index.jsx

```diff
@@ -10,17 +12,45 @@ import jetpackManageIcon from './jetpack-manage.svg';
  */
 const JetpackManageBanner = props => {
 	const { isAgencyAccount } = props;
+	const { recordEvent } = useAnalytics();
 
-	// Set up the first CTA
-	const ctaLearnMoreLabel = __( 'Learn more', 'jetpack-my-jetpack' );
-	const ctaLearnMoreUrl = getRedirectUrl( 'my-jetpack-jetpack-manage-learn-more' );
+	// Handle click events
+	const bannerClickHandler = useCallback(
+		target => {
+			recordEvent( 'jetpack_myjetpack_manage_banner_click', {
+				target: target,
+				feature: 'manage',
+			} );
+		},
+		[ recordEvent ]
+	);
+
+	const learnMoreClick = useCallback( () => {
+		bannerClickHandler( 'jp-manage-learn-more' );
+	}, [ bannerClickHandler ] );
+	const dashboardSitesClick = useCallback( () => {
+		bannerClickHandler( 'jp-manage-dashboard-sites' );
+	}, [ bannerClickHandler ] );
+	const signUpClick = useCallback( () => {
+		bannerClickHandler( 'jp-manage-sign-up' );
+	}, [ bannerClickHandler ] );
+
+	// Set up the secondary CTA
+	const secondaryCtaLabel = __( 'Learn more', 'jetpack-my-jetpack' );
+	const secondaryCtaURL = getRedirectUrl( 'my-jetpack-jetpack-manage-learn-more' );
```

Comment:
[Not a blocker] It's purely personal preference, but what if we moved this down to the return statement (`<UpsellBanner>`) ?

I personally like it because it makes it easier to read `<UpsellBanner>` since I can see what strings/functions/hooks are being passed without scrolling up and down the file.
It also makes it so only "advanced logic" exist above the return statement, which could make the component easier to glance as well